cmake/Platform/Windows/Packaging/Tests/conftest.py

```diff
@@ -0,0 +1,141 @@
+        os.environ['HOME'] = str(home_path) 
+        os.environ['HOMEPATH'] = os.environ['HOME'] 
+
+        self.installer_path = self._get_local_installer_path(request.config.getoption("--installer-uri"))
+        self.install_root = Path(request.config.getoption("--install-root")).resolve()
+        self.project_path = Path(request.config.getoption("--project-path")).resolve()
+
+        self.cmake_runtime_path = self.install_root / 'cmake/runtime'
+        self.engine_bin_path = self.install_root / 'bin/Windows/profile/Default'
+        self.project_build_path = self.project_path / 'build/Windows'
+        self.project_bin_path = self.project_build_path / 'bin/profile'
+
+        # start a log reader thread to print the output to screen
+        self.log_reader = io.open(self.temp_file.name, 'r', buffering=1)
+        self.log_reader_thread = threading.Thread(target=self._tail_log, daemon=True)
+        self.log_reader_thread.start()
+        self.log_reader_shutdown = False
+    
+    def _tail_log(self):
+        """ Tail the log file and print to screen for easy viewing in Jenkins. """
+        while True:
+            line = self.log_reader.readline()
+            if line == '':
+                if self.log_reader_shutdown:
+                    return
+            else:
+                print(line, end='')
+
+    def _get_local_installer_path(self, uri):
+        """ Return the local path to the installer, downloading the remote file if necessary """
+        parsed_uri = urllib.parse.urlparse(uri)
+        if parsed_uri.scheme in ['http', 'https', 'ftp', 'ftps', 's3']:
+            installer_path = Path(self.temp_dir) / 'installer.exe'
+
+            if parsed_uri.scheme == 's3':
+                session = boto3.session.Session()
+                client = session.client('s3')
+                client.download_file(parsed_uri.netloc, parsed_uri.path.lstrip('/'), str(installer_path))
+            else:
+                urllib.request.urlretrieve(parsed_uri.geturl(), installer_path)
+
+            # verify with SignTool if available
+            self._verify(installer_path)
+
+            return installer_path
+        else:
+            # strip the leading slash from the file URI 
+            return Path(parsed_uri.path.lstrip('/')).resolve()
+
+    def _verify(self, file):
+        """ Assert if signtool.exe exists and cannot verify the given file. """
+        try:
+            signtool_path_result = run(['powershell','-Command',"(Resolve-Path \"C:\\Program Files (x86)\\Windows Kits\\10\\bin\\*\\x64\" | Select-Object -Last 1).Path"], text=True, capture_output=True)
+            if signtool_path_result.returncode == 0:
+                signtool_path = Path(signtool_path_result.stdout.strip()) / 'signtool.exe'
+                signtool_result = self.run([str(signtool_path),'verify', '/pa', str(file)])
+                assert signtool_result.returncode == 0
+        except FileNotFoundError as e:
+            pass
```

Comment:
How about an informational or debug log about not being able to find the signtool in the except block.
That  cold help with debuggin one day.